src/accessories/characteristics/brightness.ts

```diff
@@ -41,7 +42,13 @@ export class BrightnessCharacteristic extends TuyaWebCharacteristic {
     callback: CharacteristicSetCallback
   ): void {
     // Set device state in Tuya Web API
-    const value = ((homekitValue as number) / 10) * 9 + 10;
+    const dimmerPercentage = (homekitValue as number) / 100;
+    const correctionFactor = 10;
+    const value = Math.round(
+      (100 - correctionFactor) * dimmerPercentage +
+        correctionFactor +
+        Number.EPSILON
```

Comment:
Why did you opt to add the `Number.EPSILON` here (and a few lines later)?

Reply:
The decimal number if very large, I wanted to be sure as much as possible the round will be correct